nsxt/resource_nsxt_policy_ipsec_vpn_session.go

```diff
@@ -0,0 +1,482 @@
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionComplianceSuite, false),
+				Optional:     true,
+			},
+			"connection_initiation_mode": {
+				Type:         schema.TypeString,
+				Description:  "Compliance suite.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionConnectionInitiationMode, false),
+				Optional:     true,
+				Default:      "INITIATOR",
+			},
+			"authentication_mode": {
+				Type:         schema.TypeString,
+				Description:  "Peer authentication mode. PSK - In this mode a secret key shared between local and peer sites is to be used for authentication. The secret key can be a string with a maximum length of 128 characters. CERTIFICATE - In this mode a certificate defined at the global level is to be used for authentication.",
+				ValidateFunc: validation.StringInSlice(IPSecVpnSessionAuthenticationMode, false),
+				Optional:     true,
+				Default:      "PSK",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Description: "Enable/Disable IPSec VPN session.",
+				Optional:    true,
+				Default:     true,
+			},
+			"dpd_profile_path": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing Dead Peer Detection (DPD) profile. Default is set to system default profile.",
+				Optional:    true,
+				Default:     "/infra/ipsec-vpn-dpd-profiles/nsx-default-l3vpn-dpd-profile",
+			},
+			"psk": {
+				Type:        schema.TypeString,
+				Description: "IPSec Pre-shared key. Maximum length of this field is 128 characters.",
+				Optional:    true,
+			},
+			"peer_id": {
+				Type:        schema.TypeString,
+				Description: "Peer ID to uniquely identify the peer site. The peer ID is the public IP address of the remote device terminating the VPN tunnel. When NAT is configured for the peer, enter the private IP address of the peer.",
+				Optional:    true,
+			},
+			"peer_address": {
+				Type:        schema.TypeString,
+				Description: "Public IPV4 address of the remote device terminating the VPN connection.",
+				Optional:    true,
+			},
+			"tunnel_profile_path": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing Tunnel profile to be used. Default is set to system default profile.",
+				Required:    true,
+			},
+			"local_endpoint_path": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing Local endpoint.",
+				Optional:    true,
+			},
+			"ike_profile_path": {
+				Type:        schema.TypeString,
+				Description: "Policy path referencing IKE profile.",
+				Optional:    true,
+			},
+			"tier0_id": {
+				Type:        schema.TypeString,
+				Description: "Unique identifier of the T0 resource.",
+				Optional:    true,
+				Default:     "vmc",
```

Comment:
This is not VMC-specific resource, so VMC should not appear in default
As above, lets consider `locale_service_path` here